validations/src/tests.rs

```diff
@@ -1892,7 +1892,7 @@ fn commitment_proof_lower_than_target() {
     let x = test_commit_difficult_proof();
     // This is just the hash of the VRF, we do not care for the exact value as
     // long as it is below the target hash
-    let vrf_hash = "b72c291933558248d6827d992dee7b3e0926950e0e23342f0a8e38a114ef9f81"
+    let vrf_hash = "496449cc6bcaa7210e56cedc3b0cb1075bd3ca2c693c6269620423895d4e2c71,"
```

Comment:
We should remove this hash from this test, for example replace the `assert_eq!` with a match and ignore the `vrf_hash` field.